src/EnergyPlus/WindowAC.cc

```diff
@@ -240,8 +240,6 @@ namespace WindowAC {
         Array1D_int OANodeNums(4);       // Node numbers of Outdoor air mixer (OA, EA, RA, MA)
         int IOStatus;                    // Used in GetObjectItem
         bool ErrorsFound(false);         // Set to true if errors in input, fatal at end of routine
-        bool errFlag(false);             // Local error flag for GetOAMixerNodeNums
-        bool FanErrFlag(false);          // Error flag used in GetFanIndex call
```

Comment:
[src/EnergyPlus/WindowAC.cc:243]:(style),[variableScope],The scope of the variable 'errFlag' can be reduced.
[src/EnergyPlus/WindowAC.cc:244]:(style),[variableScope],The scope of the variable 'FanErrFlag' can be reduced.
[src/EnergyPlus/WindowAC.cc:243]:(style),[unreadVariable],Variable 'errFlag' is assigned a value that is never used.